.github/actions/build/action.yml

```diff
@@ -29,3 +29,8 @@ runs:
     - name: Test Python
       shell: bash
       run: make test-py
+    - name: Build Publishable release distributions for PyPI
+      shell: bash
+      run: |
+        python -m pip install build
+        python -m build
```

Comment:
This isn't using any of the previous steps here, we'll need to pass cmake config options to build -> cmake similar to what I describe here in #41

Reply:
This is the same build instructions as running `pip install -e .` except it delivers the packages into ./dist instead of some temp directory

Reply:
You're right that it's equivalent to `pip install .`, but that's not what we're doing (or want to be doing) in this workflow. In this workflow we configure and build directly with `cmake`, _then_ run `pip install ".[dev]" --no-build-isolation` which tells `pip` to run the compilation as a rebuild rather than a fresh build. Without `--no-build-isolation`, it's a clean build every time so all the configuring we did, gets thrown out.